src/resources/views/components/g-recaptcha.blade.php

```diff
@@ -0,0 +1,21 @@
+@props(['action'])
+<div x-data="gRecaptcha" x-init="$nextTick(()=>getResponseToken())">
+    <input type="hidden"  {{$attributes}}>
+</div>
+
+<script>
+    document.addEventListener('alpine:init', () => {
```

Comment:
doesn't this trigger fetching token once when page is loaded? if there are validation errors, you need to fetch token again. which means you need to fetch token each time when submitting. i may be wrong.

Reply:
this is what i came up with
https://gist.github.com/keevitaja/d9667072bbd7f80cd78d50259fc858ab

Reply:
@keevitaja  you are correct and i ended up implementing a solution locally for the project as this repo looks dead. but your solution sounds good